Tasks/Podvornaya/Task4/Task4.py

```diff
@@ -0,0 +1,42 @@
+while True:
+    line = input("Insert your line length:")
+    if line.isdigit():
+        line = int(line)
+        if line < 35:
+            print("Your input is incorrect. The line length must be more than 35 symbols.")
+            continue
+        else:
+            break
+    else:
+        print("Your input is incorrect. Please submit a digit.")
+        continue
+
+with open("txt.txt", "r", encoding = 'utf-8') as f:
+    text = f.readlines()
```

Comment:
reading the whole file in memory is not the best approach since it can be very big